consensus/enclave/api/src/lib.rs

```diff
@@ -39,26 +40,26 @@ pub struct LocallyEncryptedTx(pub Vec<u8>);
 #[derive(Clone, Debug, Default, Deserialize, Eq, Hash, Ord, PartialEq, PartialOrd, Serialize)]
 pub struct WellFormedEncryptedTx(pub Vec<u8>);
 
-/// Tx data we wish to expose to untrusted from well-formed Txs
-#[derive(Clone, Debug, Default, Deserialize, Eq, Hash, Ord, PartialEq, PartialOrd, Serialize)]
+/// Tx data we wish to expose to untrusted from well-formed Txs.
+#[derive(Clone, Debug, Default, Deserialize, Eq, Hash, PartialEq, Serialize)]
 pub struct WellFormedTxContext {
-    /// Tx hash.
-    tx_hash: TxHash,
-
     /// Fee included in the tx.
-    fee: u64,
+    pub fee: u64,
+
+    /// Tx hash.
+    pub tx_hash: TxHash,
 
     /// Tombstone block.
-    tombstone_block: u64,
+    pub tombstone_block: u64,
```

Comment:
Why were all of these made `pub`? They were originally not `pub` intentionally to prevent them getting changed by mistake somewhere. There are accessors for all of them below.

Reply:
They were made `pub` so that instances of WellformedTxContext can be directly initialized, which simplifies testing. Previously, we did things like WellFormedTxContext::from(tx), which involves the more complicated task of creating a Tx.

I will replace the `pub`s with a constructor, which will also work to keep the unit tests clean.